### PR TITLE
fix: about spelling and command

### PR DESCRIPTION
**Pull request by liuliuOD**
1. modify commands like docker compose -> docker-compose
2. spell like 'Hash' -> 'HASH'
3. spelling in markdown

### CODE REVIEW

File: examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb [hunk 1]
```diff
@@ -432,7 +442,7 @@
    "source": [
     "## Load Documents into the Index\n",
     "\n",
-    "Now that we have a search index, we can load documents into it. We will use the same documents we used in the previous examples. In Redis, either the Hash or JSON (if using RedisJSON in addition to RediSearch) data types can be used to store documents. We will use the HASH data type in this example. The below cells will show how to load documents into the index."
+    "Now that we have a search index, we can load documents into it. We will use the same documents we used in the previous examples. In Redis, either the HASH or JSON (if using RedisJSON in addition to RediSearch) data types can be used to store documents. We will use the HASH data type in this example. The below cells will show how to load documents into the index."
```
Comment (ted-at-openai) [inline examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb:445]: 'hash' appears to be the correct Redis spelling: https://redis.io/docs/data-types/

File: examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb [hunk 2]
```diff
@@ -682,7 +692,7 @@
     "\n",
     "``HNSW`` will take longer to build and consume more memory for most cases than ``FLAT`` but will be faster to run queries on, especially for large datasets.\n",
     "\n",
-    "The following cells will show how to create an ``HNSW`` index and run queries with it using the same data as before."
+    "The following cells will show how to create a ``HNSW`` index and run queries with it using the same data as before."
```
Comment (ted-at-openai) [inline examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb:695]: Change not needed: 'an' before acronym with a vowel sound is fine.

File: examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb (hunk 1 above)
Comment (liuliuOD) [reply examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb:445]: Thank you for your input. Actually, here I want to keep the case consistent, as you can see the same word is all in uppercase letters in this line.

File: examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb (hunk 2 above)
Comment (liuliuOD) [reply examples/vector_databases/redis/getting-started-with-redis-and-openai.ipynb:695]: Thank you. I should correct this.

**Review by arifhossain512 — APPROVED** [review]

**Comment by liuliuOD** [comment]
@arifhossain512 @ted-at-openai May I give any support?

**Review by colin-openai — APPROVED** [review]
Minor grammatical changes, happy to merge these - get the point about Hash vs. HASH, as long as it is consistent with the other datatype JSON (which it is) I reckon we can live with it.

**Pull request merged**